static/product-designer-portfolio/index.html

```diff
@@ -0,0 +1,285 @@
+          >
+            <a href="#">Home</a>
+          </li>
+          <li
+            class="relative mb-4 w-fit text-white before:absolute before:-bottom-[3px] before:left-1/2 before:h-[2px] before:w-0 before:-translate-x-1/2 before:-translate-y-1/2 before:rounded-xl before:bg-white before:duration-500 before:content-[''] hover:before:w-full lg:mb-0"
+          >
+            <a href="#work">Work</a>
+          </li>
+          <li
+            class="relative w-fit text-white before:absolute before:-bottom-[3px] before:left-1/2 before:h-[2px] before:w-0 before:-translate-x-1/2 before:-translate-y-1/2 before:rounded-xl before:bg-white before:duration-500 before:content-[''] hover:before:w-full"
+          >
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- ==== End Header ==== -->
+
+    <!-- ==== Start Hero ===  -->
+    <div id="hero">
+      <div
+        class="container mt-4 flex flex-col flex-col-reverse justify-evenly lg:mt-0 lg:h-[min(90vh,_800px)] lg:flex-row lg:items-center lg:justify-between"
+      >
+        <!-- section  -->
+        <section class="h-fit text-center lg:max-w-[40%] lg:text-left">
+          <h1 class="text-light-white mb-5 text-4xl">
+            Hi, I am Your Name A Product Designer based in City.
+          </h1>
+          <p class="text-light-gray mb-12 text-lg">
+            I help businesses and companies reach their goals by designing
+            user-centric digital products & interactive experiences.
+          </p>
+          <a
+            href="mailto:your@email.com"
+            class="bg-btn mx-auto block w-fit px-8 py-4 text-white duration-300 hover:shadow-xl hover:shadow-white/20 lg:mx-0"
+          >
+            <i class="bi bi-envelope"></i>
+            your@email.com
+          </a>
+        </section>
+
+        <!-- image  -->
+        <div class="lg:grid lg:min-w-[50%] lg:place-items-center">
+          <img
+            src="https://www.asmm.fr/wp-content/uploads/2020/09/cesar-rincon-XHVpWcr5grQ-unsplash-300x300.jpg"
+            class="w-100 mx-auto block aspect-square h-auto rounded-full"
+            alt="it's me"
+          />
+        </div>
+      </div>
+    </div>
+    <!-- ==== End Hero ===  -->
+
+    <!-- ==== Start Featured Work ====  -->
+    <section class="my-12" id="work">
+      <div class="container mb-12">
+        <h1
+          class="text-light-white before:bg-light-white relative mb-6 pl-3 text-lg font-semibold before:absolute before:left-0 before:h-full before:w-[3px] before:rounded-[5px] before:content-[''] after:absolute after:-left-[3.8px] after:bottom-0 after:h-[10px] after:w-[10px] after:-rotate-45 after:border-[3px] after:border-t-0 after:border-r-0 after:content-['']"
+        >
+          Featured Work
+        </h1>
+        <!-- gallery -->
+        <div class="grid md:grid-cols-2">
+          <div class="group relative m-2 max-h-[600px]">
+            <img
```

Comment:
To improve site performance you can use `<img src="image.jpg" alt="..." loading="lazy">`  for images under the fold (not visible on initial page load)

The loading="lazy" will instruct the browser to load images only when needed.